library/common/types/c_types.h

```diff
@@ -31,11 +31,11 @@ typedef enum { ENVOY_UNDEFINED_ERROR, ENVOY_STREAM_RESET } envoy_error_code_t;
 
 /**
  * Networks classified by last physical link.
- * ENVOY_GENERIC is default and includes cases where network characteristics are unknown.
- * ENVOY_WLAN includes WiFi and other local area wireless networks.
- * ENVOY_WWAN includes all mobile phone networks.
+ * ENVOY_NET_GENERIC is default and includes cases where network characteristics are unknown.
```

Comment:
Do you think `ENVOY_NET_DEFAULT` makes more sense here?

Reply:
@junr03 and I discussed and `DEFAULT` was one option we considered. We landed on `GENERIC` because maybe it implies it covers more than just the default case where no information is available (i.e. it also covers the general case - any network for which specific handling isn't required). I don't have strong feelings though, and would be fine with either.